recipes/double-conversion/config.yml

```diff
@@ -0,0 +1,3 @@
+versions:
+  "3.1.4":
+    folder: all
```

Comment:
```suggestion
    folder: all
  "3.1.5":
    folder: all
```

Reply:
I would go so far and say that only adding 3.1.5 should be enough